src/calendar/styles.scss

```diff
@@ -0,0 +1,170 @@
+/*
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+@use '../internal/styles/index' as styles;
+@use '../internal/styles/tokens' as awsui;
+@use '../internal/hooks/focus-visible' as focus-visible;
+@use '../internal/styles/typography/index' as typographyConstants;
+@use './motion';
+
+$calendar-header-color: awsui.$color-text-dropdown-item-default;
+$calendar-grid-border-color: awsui.$color-border-calendar-grid;
+$calendar-grid-day-color: awsui.$color-text-dropdown-item-default;
+$calendar-grid-day-name-color: awsui.$color-text-calendar-month;
+$calendar-grid-disabled-day-color: awsui.$color-text-dropdown-item-disabled;
+$calendar-grid-nonmonth-day-color: awsui.$color-text-dropdown-item-secondary;
+$calendar-grid-hover-background-color: awsui.$color-background-dropdown-item-hover;
+$calendar-grid-hover-border-color: awsui.$color-border-dropdown-item-hover;
+$calendar-grid-today-background-color: awsui.$color-background-calendar-today;
+$calendar-grid-selected-background-color: awsui.$color-background-dropdown-item-selected;
+$calendar-grid-selected-border-color: awsui.$color-border-dropdown-item-selected;
+$calendar-grid-border: 1px solid $calendar-grid-border-color;
+
+.root {
+  /* used in test-utils */
+}
+
+.calendar {
+  display: block;
+  // IE11 does not calculate the height correctly when in nested flex containers (@see https://github.com/philipwalton/flexbugs#flexbug-3)
```

Comment:
Do we still need it?

Reply:
Probably not, but I would not touch it in the scope of this PR.